src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -65,19 +67,26 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
             getNote(id, startDate);
             $scope.hasSlotChanged=false;
             $scope.hasNoteChanged=false;
-        }
+        };
 
         $scope.changeSlotStatus = function() {
+            if($scope.relativeDayNumber<0 && !isRecruiter){
+                if(!isNotificationShown) {
+                    Notification.error({message: 'You cannot edit slots from past weeks!', delay: 2000});
+                }
+                isNotificationShown=!isNotificationShown;
```

Comment:
As discussed, please check try to use only one handler for `changeSlotStatus` - either click or mousedown. Then we can remove the `isNotificationShown` and corresponding if condition.